resources/gen_hips.yaml

```diff
@@ -0,0 +1,29 @@
+description: Run GenerateHipsTask
+instrument: lsst.obs.lsst.LsstCamImSim
+tasks:
+  generateHips:
+    class: lsst.pipe.tasks.hips.GenerateHipsTask
+    config:
+      python: |
+        config.properties.creator_did_template = "temp://lsst/ci_imsim/hips/images/band_{band}"
+        config.properties.obs_title_template = "CI imSim for band {band}"
+        config.properties.obs_description_template = "Coadded data from ci_imsim, band {band}."
```

Comment:
Is there any way to test that any of the values from this config made it into a HiPS output file?